src/Context/StreamApp.js

```diff
@@ -13,7 +13,6 @@ import type { FeedProps } from './Feed';
 import { handleError } from '../errors';
 
 export const StreamContext = React.createContext({
-  changedUserData: () => {},
```

Comment:
Why are you removing this? It would be a breaking change, so if not needed can you revert that?